### PR TITLE
PTV-1908 Attempt at Text Updates and 500 limit

### PR DESCRIPTION
**Pull request by bio-boris**
Possible Text Updates

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
 # Hipmer release notes
 =========================================
 
+2.4.0
+-----
+* added an authorized user list which can exceed the input size limit of 500 Gbp
+
+2.3.0
```
Comment (bio-boris) [inline RELEASE_NOTES.md:8]: I think we can keep the version at 2.3.0, since it was never promoted to beta or released

Comment (bio-boris) [reply RELEASE_NOTES.md:8]: @mikacashman I think we should add notes about https://github.com/kbaseapps/kbase_hipmer/pull/17/files

Comment (mikacashman) [reply RELEASE_NOTES.md:4]: That is fine with me, that makes this in-line with the ui listed versions as well.  @scanon you good with the release notes from your PR to update to Perlmutter?

Comment (bio-boris) [reply RELEASE_NOTES.md:4]: @mikacashman your updates seem good enough to me! Can we please get this merge in and registered on ci and appdev and then tested on appdev when you get a chance? Thanks

**Pull request merged**